.mergify.yml

```diff
@@ -0,0 +1,60 @@
+pull_request_rules:
+  - name: backport patches to cryostat-v2.4 branch
+    conditions:
+      - base=main
+      - label=backport
+    actions:
+      backport:
+        branches:
+          - cryostat-v2.4
+        assignees:
+          - "{{ author }}"
+
+  - name: auto label PRs from reviewers
+    conditions:
+      - author=@reviewers
+    actions:
+      label:
+        add:
+          - safe-to-test
+
+  - name: dismiss stale reviews
+    conditions:
+      - base=main
+    actions:
+      dismiss_reviews:
+        approved: [ 'mergify', 'Mergifyio' ]
+        changes_requested: [ 'mergify', 'Mergifyio' ]
+
+  - name: block pull requests that manually update web-client submodule
+    conditions:
+      - or:
+        - files=src/main/webui
+        - files~=src/main/webui/
+    actions:
+      review:
+        type: REQUEST_CHANGES
+        message: >-
+            Pull Request blocked. `web-client` submodule updates are performed automatically by CI when that repository
```

Comment:
This should probably reference `src/main/webui` since that is the new path equivalent to the old `web-client`.